docs/api-and-integration/octo.exe-command-line/creating-releases.md

```diff
@@ -149,18 +149,25 @@ octo create-release --project HelloWorld --version 1.0.3 --server http://octopus
 
 ## Specifying the package version {#Creatingreleases-Specifyingthepackageversion}
 
+The step name and version pair will take the highest precedence supplied via `--package stepName:1.0.3`, if that version cannot be found then the version supplied by the `--packageversion` will be next, finally falling back to the what is found specified as specified by `--packagesFolder`.
```

Comment:
Not quite right. Also the --packagesFolder parameter may override an earlier parameter if specified last.

It's more like: 
For each step that has a package, the version is determined in the following order:
1) The step name matches a `--packageVersion` parameter or a file filename found by `--packageFolder`
2) The package id matches a `--packageVersion` parameter or a file found by `--packageFolder`
3) The value from the `--package` parameter

If there are duplicate names/ids as a result of the `--packageVersion` and `--packageFolder` parameters, the last one specified is used.